### DISCUSSION TITLE
[Feature] backup and restore

### DISCUSSION THREAD

**Discussion by dkerlee** [immich-app/immich#7549] [Feature Request]
### The feature

backup and restore GUI system.

As it stands at the moment, this would include 
1) asset files (jpg, heic, mov, etc)
2) postgresql DB

restoring should also be GUI based at the beginning of a new install: is this a new install, or restore from backup?

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by dkerlee** [comment]
I have several ideas here. What about cross internet Immich buddy who has a similar sized library, and similar addition frequency, and we just swap encrypted bits anonymously. We’d each get a report on a dashboard of what’s where, how much GB has flown higher and thither.

**Comment by dkerlee** [comment]
Simply plug ish drive into your machine, set a mount point and tell Immich put backup files right there. Users can make additional offsite copies via rsync, Dropbox, or some other backup scheme.

**Comment by dkerlee** [comment]
I’m sure there must be a reason for requiring a full external PostgreSQL database for this. But, what kind of speed or install size hot would we take if some version of PostgreSQL (or whatever) was just included with this docker? Backup and restore could be much simpler if you had control of the entire database machine right there.

**Comment by bo0tzz** [comment]
Duplicate of #5441
